src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private static final Random random = new Random(47);
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
Avoid using static methods or static variables unless they are constants. Here, you should create instances of `Random` and `ColorSupplier` in the constructor or as instance variables without the `static` keyword. This will also help in making your class more testable because you can mock these dependencies if needed.